static/src/hud.js

```diff
@@ -1,7 +1,29 @@
 import { clamp01, toLogScale01 } from "./math.js"
 import { SendChannel, RecieveChannel } from "./channel.js"
 
-const MIDI_MAPPING_STR = '[["r1c1","s16"],["r1c2","s20"],["r1c3","s24"],["r1c4","s28"],["r1c5","s46"],["r1c6","s50"],["r1c7","s54"],["r1c8","s58"],["r2c1","s17"],["r2c2","s21"],["r2c3","s25"],["r2c4","s29"],["r2c5","s47"],["r2c6","s51"],["r2c7","s55"],["r2c8","s59"],["r2c9","b25"],["r3c1","s18"],["r3c2","s22"],["r3c3","s26"],["r3c4","s30"],["r3c5","s48"],["r3c6","s52"],["r3c7","s56"],["r3c8","s60"],["r3c9","b26"],["r4c1","b1"],["r4c2","b4"],["r4c3","b7"],["r4c4","b10"],["r4c5","b13"],["r4c6","b16"],["r4c7","b19"],["r4c8","b22"],["r4c9","b27"],["r5c1","b3"],["r5c2","b6"],["r5c3","b9"],["r5c4","b12"],["r5c5","b15"],["r5c6","b18"],["r5c7","b21"],["r5c8","b24"],["r6c1","s19"],["r6c2","s23"],["r6c3","s27"],["r6c4","s31"],["r6c5","s49"],["r6c6","s53"],["r6c7","s57"],["r6c8","s61"],["r6c9","s62"]]'
+const MIDI_MAPPING_STR = '[["r1c1","s16"],\
```

Comment:
Cleaned up the mapping for the MidiMix - set the top row to be jets, the bottom row to be grid, and the sliders to be the more powerful and configy-params. This way - it's generally safer to turn a random knob and it's also easier to navigate